src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -3770,3 +3766,17 @@ class FileSegment(BaseFileSegment):
             allow_trailing=True,
         ),
     )
+
+
+@exasol_dialect.segment()
+class EmitsGrammar(BaseFileSegment):
+    """EMITS Grammar for JSON_EXTRACT for example.
+
+    In it's own class to give it a type to allow L013 to find it easily.
```

Comment:
```suggestion
    In it's own segment to give it a type to allow L013 to find it easily.
```

Reply:
OK makes sense.

Though would be nice to be able to add a name attribute to Grammars without having to "upgrade" Grammars Segments just for that 😞